client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -690,6 +692,19 @@ static Request explainLifecycle(ExplainLifecycleRequest explainLifecycleRequest)
         return request;
     }
 
+    static Request retryLifecycle(RetryLifecyclePolicyRequest retryLifecyclePolicyRequest) {
```

Comment:
These should be in some IndexLifecycleRequestConverters class. i dont think it exists yet.